pipeline/enigma/README.md

```diff
@@ -6,39 +6,39 @@ raw_files/ENIGMA_for_BRCAsite_9.21.2015.xlsx
 
 raw_files/ENIGMA_SubmissionClinVar_2016-05-31_ncbi.xlsx
```

Comment:
It looks like this file as well as the enigma-processing.py file are out of sync with master. They could both use review to determine which is more correct.